axe.d.ts

```diff
@@ -4,36 +4,54 @@
 
 declare module axe {
 
-	type ImpactValue = "minor" | "moderate" | "serious" | "critical";
+	enum ImpactValue {
+		minor = "minor",
+		moderate = "moderate",
+		serious = "serious",
+		critical = "critical"
+	}
 
-	type TagValue = "wcag2a" | "wcag2aa" | "section508" | "best-practice";
+	enum TagValue {
+		wcag2a = "wcag2a",
```

Comment:
This will break users' code if they were previously using `TagValue`s.

For example, we're no longer able to call [`axewebdriverjs#withTags(['wcag2a'])`](https://github.com/dequelabs/axe-webdriverjs/blob/ce8d9acc83f0b4d3c52f5df557c3561259e00469/lib/index.js#L72), but instead, will need to do `axewebdriverjs#withTags([ axe.TagValue.wcag2a ])`.

Are we comfortable making a breaking change here?